scripts/dump-schema.js

```diff
@@ -0,0 +1,30 @@
+import fs from 'fs';
+import path from 'path';
+import schema from '../schema';
+import { graphql }  from 'graphql';
+import { introspectionQuery, printSchema } from 'graphql/utilities';
+
+const destination = process.argv[2];
+if (destination === undefined || !fs.existsSync(destination)) {
+  console.error('Usage: dump-schema.js /path/to/output/directory');
```

Comment:
You could either use the `debug` module or simply disable lint for this line/rule